botorch/models/contextual_multioutput.py

```diff
@@ -8,6 +8,8 @@
 
 import torch
 from botorch.models.multitask import MultiTaskGP
+from botorch.models.transforms.outcome import OutcomeTransform
+from botorch.models.transforms.input import InputTransform
```

Comment:
Internal linter complaining here. I will add a  µsort step to CI soon so this is more obvious. 

```suggestion
from botorch.models.transforms.input import InputTransform
from botorch.models.transforms.outcome import OutcomeTransform
```